apps/web-portal/package.json

```diff
@@ -10,16 +10,16 @@
   "scripts": {
     "start": "node --experimental-modules --es-module-specifier-resolution=node .",
     "start:debug": "nodemon --watch ../../packages --watch ./lib --experimental-modules --es-module-specifier-resolution=node --inspect .",
-    "test": "echo \"No tests yet, need to add api integration tests\"",
+    "test": "ava --concurrency 1",
```

Comment:
Extend `tth` to use instead of direct call for  `ava`

also it's might be related to changes in `tests.yml`

```

      - name: Tests
        env:
          NODE_OPTIONS: '--experimental-modules --es-module-specifier-resolution=node'

```
not clear, because that option is defined in `ava.config.cjs` ...

Reply:
extracted to sep issue